bazel/envoy_mobile_copts.bzl

```diff
@@ -0,0 +1,8 @@
+# DO NOT LOAD THIS FILE. Load envoy_build_system.bzl instead.
+load(":envoy_select.bzl", "envoy_select_admin_functionality", "envoy_select_enable_http3", "envoy_select_envoy_mobile_request_compression")
+
+# Compute the copts for Envoy mobile.
```

Comment:
we have a lot of copts for mobile which aren't represented here.  Can you comment which ones matter? is it just ones for APIs which are compiled in and out or should we actually replicate more of the select/defines that Envoy has?

Reply:
Oh sorry, we need to plumb these through to `objc_library` and `swift_library` so since we need this in 3 places, I think it's best to keep centralized.